### PR TITLE
Fixes to tutorial notebooks

### PR DESCRIPTION
**Pull request by adelpierre**
Added support to non-CUDA devices to `norse_tutorial`
Changed some comments on `spiking_introduction`

### CODE REVIEW

File: spiking_introduction.ipynb [hunk 1]
```diff
@@ -56,38 +56,58 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Creating 10 leaky integrate-and-fire (LIF) neurons\n",
+    "## Creating leaky integrate-and-fire (LIF) neurons\n",
     "\n",
-    "To start simulating we need `10` neurons. Each neuron needs to be set to a default state (`lif_state`). This default state tells us what the initial neuron parameters are (for now, we are just using the defaults), how many batches we have (`1`, so none), and what device we will use to simulate our neuron on (`cpu`).\n",
+    "To start simulating we need to create a population of LIF neurons of an arbitrary size. Each neuron in the population is set to a default state. This default state tells us what the initial neuron parameters are (for now, we are just using the defaults). You can find more informations [here](https://norse.github.io/norse/auto_api/norse.torch.functional.lif.html#norse.torch.functional.lif.LIFParameters).\n",
```
Comment (Jegp) [inline spiking_introduction.ipynb:61]: Nice reference! :)

**Review by Jegp — CHANGES_REQUESTED** [review]
Great work Adrien. I admire your eye for detail. You've linked to documentation and clarified the text greatly, so this is a nice contribution.

I also like that you included pictures, so people can actually just skim through it if they don't want to do the notebook.

There's one problem in the norse tutorial notebook, where one of the cells fail because it was interrupted. That should be run more efficiently.

Tomorrow, we can perhaps look at fixing your CUDA drivers. And then I can give you access to a machine with much more GPU power that should be helpful.

**Comment by Jegp** [comment]
Any news on this @adelpierre ?

**Comment by adelpierre** [comment]
Should be fixed with the latest commit !

**Comment by Jegp** [comment]
Nice work :-)

**Pull request merged**